src/pollution_model/run_pollution_model.py

```diff
@@ -0,0 +1,454 @@
+        attributes (Index, SurfaceArea, SurfaceType)
+    """
+    # Select all relevant information from the appropriate table
+    # TODO: Update this when the database has been created
+    query = text("SELECT building_id FROM nz_building_outlines")
+    # Execute the SQL query
+    result = engine.execute(query).fetchall()
+    new_result = []
+    for item in result:
+        # TODO: Update this when the real database has been created
+        # Append appropriate attribute data to the list. The attributes are Index, SurfaceArea, and SurfaceType.
+        # Additionally, a placeholder for TSS, TCu, TZn, DCu, and DZn are included and set to "None". These will be
+        # edited later.
+        new_result.append([item[0], 1, 1, None, None, None, None, None])
+    # Convert the list into a numpy array
+    np_data = np.array(new_result)
+    # Define the names of columns in the xarray
+    col_names = ["Index", "SurfaceArea", "SurfaceType", "TSS", "TCu", "TZn", "DCu", "DZn"]
+    # return the xarray containing the relevant data about buildings
+    return xr.DataArray(new_result, dims=("row", "col"), coords={'row': range(np_data.shape[0]), 'col': col_names})
+
+
+def get_road_information(engine: Engine):
+    """
+    Extracts relevant information about roads and car parks from the database and formats them such that they are easy
+    to use for pollution modeling purposes.
+
+    Parameters
+    ----------
+    engine: Engine
+      The sqlalchemy database connection engine
+
+    Returns
+    -------
+    xr.DataArray
+        A DataArray containing rows corresponding to roads, and columns corresponding to
+        attributes (Index, SurfaceArea, SurfaceType)
+    """
+    # Select all relevant information from the appropriate table
+    # TODO: Update this when the database has been created -- Need to update the name of the table!
+    query = text("SELECT road_id FROM nz_roads")
+    # Execute the SQL query
+    result = engine.execute(query).fetchall()
+    new_result = []
+    for item in result:
+        # TODO: Update this when the real database has been created
+        # Append appropriate attribute data to the list. The attributes are Index, SurfaceArea, and SurfaceType.
+        # Additionally, a placeholder for TSS, TCu, TZn, DCu, and DZn are included and set to "None". These will be
+        # edited later.
+        new_result.append([item[0], 1, 1, None, None, None, None, None])
+    # Convert the list into a numpy array
+    np_data = np.array(new_result)
+    # Define the names of columns in the xarray
+    col_names = ["Index", "SurfaceArea", "SurfaceType", "TSS", "TCu", "TZn", "DCu", "DZn"]
+    # return the xarray containing the relevant data about buildings
+    return xr.DataArray(new_result, dims=("row", "col"), coords={'row': range(np_data.shape[0]), 'col': col_names})
+
+
+def run_pollution_model_rain_event(
+        engine: Engine,
+        area_of_interest: gpd.GeoDataFrame,

```

Comment:
the `area_of_interest` parameter is not currently been used, it should be a parameter in `get_building_information()` and `get_road_information()` for fetching buildings and roads within the catchment area.